### PR TITLE
Add P class and tests for it

### PR DESCRIPTION
**Pull request by core2duo**

### CODE REVIEW

File: trusts/decorators.py [hunk 1]
```diff
@@ -42,6 +108,7 @@ def _wrapped_view(request, *args, **kwargs):
         return _wrapped_view
     return decorator
 
+
```
Comment (thomasyip) [inline trusts/decorators.py:111]: Code format: One line to separate method, two lines to separate class.

Comment (core2duo) [inline trusts/decorators.py:111]: Well, PEP-8 doesn't agree with that:

> Surround top-level function and class definitions with two blank lines.
> Method definitions inside a class are surrounded by a single blank line.

https://www.python.org/dev/peps/pep-0008/#blank-lines
But I can delete these blank lines if you want.

Comment (thomasyip) [inline trusts/decorators.py:111]: Okay. Follows PEP-8.

I didn't know the "standard" distinguishes between top level and not top level.

File: trusts/tests.py [hunk 2]
```diff
@@ -199,7 +199,7 @@ def test_permission_required(self):
         self.assertEqual(filter.count(), 1)
         self.assertEqual(filter.first().pk, self.group.pk)
 
-        # test b) has_perms() == False
+        # test b) has_perms() == True
```
Comment (thomasyip) [inline trusts/tests.py:202]: Good catch. Thank you.

**Comment by thomasyip** [comment]
Closing this to use common repo. See PR #3.

**Pull request closed without merging**